priv/templates/phx.gen.live/show.html.heex

```diff
@@ -17,7 +17,7 @@
 <.modal :if={@live_action == :edit} id="<%= schema.singular %>-modal" show on_cancel={JS.patch(~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}")}>
   <.live_component
     module={<%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.FormComponent}
-    id={@<%= schema.singular %>.id}
```

Comment:
13085

Reply:
Any good ?

Reply:
I am a bit unclear about your comment.  The suggested change conditionally adds a prefix using either `:id` or the name of the `primary_key` if one is defined:
```elixir
-    id={@<%= schema.singular %>.id}
+    id={@<%= schema.singular %>.<%= schema.opts[:primary_key] || :id %>}
```

Reply:
I also don't see anything wrong with that line.